python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,31 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+# Maximum size of a string column is 2gb
+_STRINGS_UDF_DEFAULT_HEAP_SIZE = 2**31
+heap_size = 0
+
+
+def set_malloc_heap_size(size=None):
+    """
+    Heap size control for strings_udf, size in bytes.
+    """
+    global heap_size
+    if size == None:
+        size = os.environ.get(
```

Comment:
Generally environment variables take effect at load time. I don't think we should allow/encourage the user to modify the environment of the process while it is running in order to change this.

I think you want to define it on import:
```python
_STRINGS_UDF_DEFAULT_HEAP_SIZE = os.environ.get("STRINGS_UDF_HEAP_SIZE", 2**31)
```